include/odp/api/spec/ipsec.h

```diff
@@ -232,6 +232,12 @@ typedef struct odp_ipsec_capability_t {
 	 */
 	odp_support_t pipeline_cls;
 
+	/** Implementation will automatically truncate TFC padding in received
+	 *  packets in ESP tunnel mode. Application can use this capability to
+	 *  determine necessity for ESP_TFC_PADDING_NOT_SUPPORTED notification.
+	 */
+	odp_support_t tfc_padding_truncate;
```

Comment:
I thought we discussed removing these capabilities from Tiger Moth?

Reply:
I moved it to a separate comment, because of `ESP_TFC_PADDING_NOT_SUPPORTED` notification. I can still drop it.